docs/security/outbound-requests/telemetry.md

```diff
@@ -47,6 +47,7 @@ In addition, we'll send some specific aggregate criteria along with the request.
 | Structured Configuration Variable usage across projects including how many steps have Structured Configuration Variables enabled and what file extensions are being used | 2020.4.0 |
 | OS Architecture of Deployment Target Tentacles | 2021.1.0 |
 | The number of Projects that are version controlled | 2021.2.0 |
+| Aggregated data about the number and type of change managed deployments | 2022.2.0 |
```

Comment:
Should this mention ITSM change managed deployments. The wording without ITSM in it seems a bit general.

Reply:
@adrianhowchin thoughts?

Reply:
Updated to "...number and type of ITSM change managed..." as per @adrianhowchin